src/Aspire.Hosting/ApplicationModel/CommandsConfigurationExtensions.cs

```diff
@@ -9,6 +9,11 @@ namespace Aspire.Hosting.ApplicationModel;
 
 internal static class CommandsConfigurationExtensions
 {
+    private const int SYSTEM_COMMAND_BASE = 10000;
```

Comment:
we follow dotnet/aspnetcore repo style here, which requires PascalCase for consts.